channel-email/src/main/java/com/synopsys/integration/alert/channel/email/action/EmailGlobalTestAction.java

```diff
@@ -29,15 +39,28 @@
     private static final String TEST_SUBJECT_LINE = "Email Global Configuration Test";
     private static final String TEST_MESSAGE_CONTENT = "This is a test message from Alert to confirm your Global Email Configuration is valid.";
 
+    private final ValidationHelper validationHelper;
+    private final TestHelper testHelper;
+    private final EmailGlobalConfigurationValidator validator;
+
     private final EmailChannelMessagingService emailChannelMessagingService;
     private final JavamailPropertiesFactory javamailPropertiesFactory;
 
     @Autowired
-    public EmailGlobalTestAction(EmailChannelMessagingService emailChannelMessagingService, JavamailPropertiesFactory javamailPropertiesFactory) {
+    public EmailGlobalTestAction(TestHelper testHelper, ValidationHelper validationHelper, EmailGlobalConfigurationValidator validator,
+        EmailChannelMessagingService emailChannelMessagingService, JavamailPropertiesFactory javamailPropertiesFactory) {
+        this.testHelper = testHelper;
+        this.validationHelper = validationHelper;
+        this.validator = validator;
         this.emailChannelMessagingService = emailChannelMessagingService;
         this.javamailPropertiesFactory = javamailPropertiesFactory;
     }
 
+    public ActionResponse<ValidationResponseModel> test(String testAddress, EmailGlobalConfigModel requestResource) {
```

Comment:
Little confused by the differences between this test method and the testConfig method. Can we change the name to reflect the differences?